source/release-notes.txt

```diff
@@ -20,6 +20,67 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-14-16:
+
+|bi| 2.14.16
+------------
+
+*Released September 10th, 2024*
+
+Improvements
+~~~~~~~~~~~~
+
+- Now built with Go 1.21.12+ to address `CVE-2024-24791 
+  <https://www.cve.org/CVERecord?id=CVE-2024-24791>`__ for all 
+  operating systems, including the following operating systems:
+
+  - RHEL 7.0
+  - RHEL 8.0 zSeries
+  - Windows 64
+  - macOS 13 arm/OpenSSL 3.0
+  - Ubuntu 18.04 container
+
+- |bi| now supports the upcoming MongoDB version, 8.0.
+- Starting in MongoDB version 8.0, you can convert ``binData`` to
+  ``string`` and ``string`` to ``binData`` by using the :manual:`$convert </reference/operator/aggregation/convert/>`
+  aggregation and providing the required option to specify the format.
+  
+  Converting ``binData`` to ``string`` and ``string`` to ``binData``
+  is unsupported for MongoDB versions 7.0 and earlier, and returns ``null``.
```

Comment:
Now that I see it, I think this is too much details for the release notes.
There are no user facing changes so let's just not get into the details and remove this section.
```suggestion
```

Reply:
@nbagnard Sorry for the delay, I removed this, please take another look when you have a moment and I can merge these in. Thanks!